src/Middleware/HeaderPropagation/src/DependencyInjection/HeaderPropagationApplicationBuilderExtensions.cs

```diff
@@ -8,6 +8,9 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// <see cref="IApplicationBuilder"/> extension methods for propagating request headers to an <see cref="HttpClient"/>.
```

Comment:
Most other middleware follow the format of "Extensions method for x middleware." Ex: https://github.com/dotnet/aspnetcore/blob/master/src/Middleware/HostFiltering/src/HostFilteringBuilderExtensions.cs.

Reply:
What about 

```suggestion
    /// <see cref="IApplicationBuilder"/> extension methods for <cref="HeaderPropagationMiddleware"/> which propagates request headers to an <see cref="HttpClient"/>?
```

And then similar updates to the other HeaderPropigationWhateverExtensions classes? We're not super consistent with the format for these, and I like describing what makes the various HeaderPropigationWhateverExtensions classes different.

Reply:
Sounds good!